frontend/src/metabase-lib/lib/queries/structured/Filter.ts

```diff
@@ -72,7 +76,10 @@ export default class Filter extends MBQLClause {
       const dimensionName =
         dimension && includeDimension && dimension.displayName();
       const operatorName =
-        operator && !isStartingFrom(this) && operator.moreVerboseName;
+        operator &&
+        includeOperator &&
+        !isStartingFrom(this) &&
+        operator.moreVerboseName;
```

Comment:
because we're now showing all operators with field names, we need the option to not show them in `.displayName()`

![Screen Shot 2022-07-13 at 1 13 26 PM](https://user-images.githubusercontent.com/30528226/178826096-804a7ca9-7716-479f-8e12-ffc0b75c6940.png)